### PR TITLE
fix: files are copied parallely insdead of sequentially

### PR DESCRIPTION
**Pull request by devsargam**
This pr adds:
- When files are copied in s3 this approach uses Promise.all instead of awaiting each files to be copied'

### CODE REVIEW

**Comment by hkirat** [comment]
Already closed in #1

File: good-code/init-service/src/aws.ts [hunk 1]
```diff
@@ -34,10 +39,13 @@ export async function copyS3Folder(sourcePrefix: string, destinationPrefix: stri
             };
             console.log(copyParams)
 
-            await s3.copyObject(copyParams).promise();
+            filesToBeCopied.push(s3.copyObject(copyParams).promise());
```
Comment (hkirat) [inline good-code/init-service/src/aws.ts:42]: I like this approach better than #1 fwiw

**Comment by devsargam** [comment]
Thanks a lot Kirat 🙇🏻‍♀️

**Pull request closed without merging**